doc/sphinx/practical-tools/coq-commands.rst

```diff
@@ -42,16 +42,52 @@ toplevel with the command ``Coqloop.loop();;``.
 Batch compilation (coqc)
 ------------------------
 
-The ``coqc`` command takes a name *file* as argument. Then it looks for a
-file named *file*.v, and tries to compile it into a
-*file*.vo file (See :ref:`compiled-files`).
-
-.. caution::
-
-   The name *file* should be a regular Coq identifier as defined in Section :ref:`lexical-conventions`.
-   It should contain only letters, digits or underscores (_). For example ``/bar/foo/toto.v`` is valid,
-   but ``/bar/foo/to-to.v`` is not.
-
+The ``coqc`` command compiles a Coq proof script file with a ".v" suffix
+to create a compiled file with a ".vo" suffix.  (See :ref:`compiled-files`.)
+The components of the filename must be valid Coq identifiers as described in
+:ref:`lexical-conventions`; they should contain only letters, digits or
+underscores (_) with a ".v" suffix on the final component.
+For example ``/bar/foo/toto.v`` is valid, but ``/bar/foo/to-to.v`` is not.
+
+We recommend specifying a logical directory name (which is also the module name)
+with the `-R` or the `-Q` options.
+Generally we recommend using provided utilities such as `make` to build Coq projects.
+See :ref:`building_coq_project`.
```

Comment:
:+1:

```suggestion
We recommend specifying a logical directory name (which is also the module name)
with the `-R` or the `-Q` options.
Generally we recommend using provided utilities such as `coq_makefile` or `dune` to build Coq projects.
See :ref:`building_coq_project`.
```

Reply:
`coq_makefile` creates a makefile that you then run with `make`, right?

Made it: "Generally we recommend using utilities such as `make` or `dune` to build Coq projects,
using `coq_makefile` to generate the dependencies."

Reply:
> `coq_makefile` creates a makefile that you then run with `make`, right?

Yes.
 
> Made it: "Generally we recommend using utilities such as `make` or `dune` to build Coq projects,
> using `coq_makefile` to generate the dependencies."

Maybe tweak it further to:

"Generally we recommend using utilities such as `make` (using `coq_makefile` to generate the `Makefile`) or `dune` to build Coq projects"
> ."